hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/fs/TestSymlinkHdfs.java

```diff
@@ -100,7 +100,7 @@ public static void afterClassTeardown() throws Exception {
     if (cluster != null) {
       cluster.shutdown();
     }
-    IOUtils.closeQuietly(webhdfs);
+    IOUtils.closeStream(webhdfs);
```

Comment:
I am not sure what @steveloughran meant here:
https://github.com/apache/hadoop/pull/5342#issuecomment-1422976653
by
```
should have said use o.a.hadoop.utils.IOUtils.closeStream(), because that one logs at debug if an exception is raised
```

``IOUtils.closeStream(webhdfs);`` won't be logging either.
Check the code:
It calls ``cleanUpWithLogger``, with Logger passed as ``null``
```
  public static void closeStream(java.io.Closeable stream) {
    if (stream != null) {
      cleanupWithLogger(null, stream);
    }
```
And ``cleanUpWithLogger`` won't log because we passed ``null`` for ``logger``
```
        } catch (Throwable e) {
          if (logger != null) {
            logger.debug("Exception in closing {}", c, e);
          }
```

May be the intention was to ask: use `` IOUtils.cleanupWithLogger(LOG, webhdfs);``

Reply:
realy, I always thought closeStream() would pass in the default logger if none set. If that is *not the case* then we should fix it...it gets used in a lot of places

Reply:
Since the request was that the closing of the stream log problems, I'll go ahead and switch it to `IOUtils.cleanupWithLogger(LOG, webhdfs)`.